teuthology/util/containers.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import re
+import requests
+
+log = logging.getLogger(__name__)
+
+# Our container images use a certain base image and flavor by default; those
+# values are reflected below. If different values are used, they are appended
+# to the image name.
+DEFAULT_CONTAINER_BASE = 'centos:9'
+DEFAULT_CONTAINER_FLAVOR = 'default'
+DEFAULT_CONTAINER_IMAGE='quay.ceph.io/ceph-ci/ceph:{sha1}'
```

Comment:
doubt it matters but worth noting our current container based teuthology tests pull from `quay-quay-quay...` rather than `quay.ceph.io`